proj.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np # linear algebra
+import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)
+import matplotlib.pyplot as plt
+import seaborn as sns
+import os
+import sqlite3
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-r', '--ratings', action='store', dest='ratings', help='Shows plot for ratings.')
+parser.add_argument('-t', '--typeComparisson', action='store', dest='frequencyTR', help='Shows comparisson between type and ratings.')
+parser.add_argument('-y', '--yearAdded', action='store', dest='yearAdded', help='Shows frequency of Movies and TV Shows released in years.')
+parser.add_argument('-s', '--releaseSort', action='store', dest='releaseSort', help='Shows count of Movies and TV Shows sorted acc. to release year.')
+parser.add_argument('-c', '--countrySort', action='store', dest='countrySort', help='Shows count of Movies and TV Shows sorted acc. to countries.')
+parser.add_argument('-l', '--listedSort', action='store', dest='listedSort', help='Shows count of Movies and TV Shows sorted acc. to the year of listing.')
+
+args = parser.parse_args()
+rating = args.ratings
+freqTRD = args.frequencyTR
+year = args.yearAdded
+release = args.releaseSort
+country = args.countrySort
+listed = args.listedSort
+
+if(rating):
+  ratings()
+elif (freqTRD):
+  freqTR()
+elif (year):
+    yearAdded()
+elif(release):
+    releaseSort()
+elif(country):
+    countrySort()
+elif(listed):
+    listedSort()
+
+df = pd.read_csv(r'C:\Users\Dell\Downloads\archive\netflix_titles.csv')
```

Comment:
make your code functional, for such kind of tasks (loading data, creating cursor) create a function `init_app`.

Reply:
init_app added!